cve_bin_tool/checkers/mysql.py

```diff
@@ -28,3 +28,12 @@ class MysqlChecker(Checker):
     ]
     VERSION_PATTERNS = [r"mysql\-([0-9]+\.[0-9]+\-[0-9]+\.[0-9]+\.[0-9]+)"]
```

Comment:
What you've got will work, but wouldn't it be easier to change the pattern here?  the part we treat as a version is what's matched inside the round brackets `()` so if we want to discard the first part entirely, we could just move that first bracket:

```suggestion
    VERSION_PATTERNS = [r"mysql\-[0-9]+\.[0-9]+\-([0-9]+\.[0-9]+\.[0-9]+)"]
```

Or will that break some older parsing?

Reply:
It didn't break the parsing. I made the changes